SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -161,7 +161,9 @@ install -D -m 0755 kata-runtime %{buildroot}%{coco_bin}/kata-runtime
 install -D -m 0755 data/kata-collect-data.sh %{buildroot}%{coco_bin}/kata-collect-data.sh
 
 install -D -m 0644 config/configuration-clh.toml %{buildroot}/%{defaults_kata}/configuration-clh.toml
+install -D -m 0644 config/configuration-clh-snp.toml %{buildroot}/%{defaults_kata}/configuration-clh-snp.toml
```

Comment:
the vanilla kata package should install its own CLH config, while the CC package should install its own CLH SNP config, right?

Reply:
The vanilla kata package does not install a "-clh" config. Its config file is named configuration.toml.

Reply:
In other words: Why do we need two configurations? :)

Reply:
This allows us to have the standard kata-cc config along with the SNP config. Vanilla kata actually installs many different configs so it's similar in that sense. If we're never going to use the standard kata-cc config anymore then it can be removed, but having flexibility here might be advantageous for comparing both cases.